src/visual.ts

```diff
@@ -499,6 +513,9 @@ module powerbi.extensibility.visual {
 
                 this.behavior.renderSelection(false);
             }
+
+            this.axisX.selectAll("text").style("font-size", this.data.settings.categoryAxis.fontSize);
+            this.axisY.selectAll("text").style("font-size", this.data.settings.valueAxis.fontSize);
```

Comment:
Move it to renderYAxisLabels() method

Reply:
It's not working if put to renderYAxisLabels